fastlane/Fastfile

```diff
@@ -17,4 +17,15 @@ platform :ios do
       test | xcpretty
       ")
   end
+
+  desc "Runs SwiftLint and returns any errors if our linter rule aren't met"
+  lane :lint do
+    swiftlint(
+      mode: :lint,
+      output_file: "swiftlint.result.json",
```

Comment:
I think we would want to add this file (`swiftlint.result.json`) to our .gitignore. What do you think?

Reply:
Thinking about it now, since this currently only runs in CI, idk how helpful an `output_file` will be. I think I was thinking about this in the sense of running the fastlane script manually via the command line, so I'm thinking for now we remove this line. Thoughts?

Reply:
I'm good with removing it entirely 👍